doc/source/serve/http-guide.md

```diff
@@ -183,3 +184,16 @@ Client disconnecting
 (ServeReplica:default_StreamingResponder pid=50842) Cancelled! Exiting.
 (ServeReplica:default_StreamingResponder pid=50842) INFO 2023-07-10 16:08:45,756 default_StreamingResponder default_StreamingResponder#cmpnmF ahteNDQSWx / default replica.py:691 - __CALL__ OK 1019.1ms
 ```
+
+
+(serve-http-guide-keep-alive-timeout)=
+## Set keep alive timeout
+
+Serve uses a Uvicorn HTTP server internally to serve HTTP requests. By default, Uvicorn
+keeps HTTP connections alive for 5 seconds between requests. Modify the keep-alive
+timeout by setting the `keep_alive_timeout_s` in the `http_options` field of the Serve
+config files. This config is global to your Ray cluster, and it cannot be updated during
+runtime. You can also set `RAY_SERVE_HTTP_KEEP_ALIVE_TIMEOUT_S` environment variable to
+set the keep alive timeout. `RAY_SERVE_HTTP_KEEP_ALIVE_TIMEOUT_S` will take the
+precedence over the `keep_alive_timeout_s` config if both are set. You can find more
+information about Uvicorn's keep alive timeout [here](https://www.uvicorn.org/settings/#keep-alive-timeout).
```

Comment:
```suggestion
Uvicorn's keep alive timeout [guide](https://www.uvicorn.org/settings/#keep-alive-timeout) for more information.
```

Reply:
I think this reference is broken. Did you mean: https://www.uvicorn.org/settings/#timeouts ?

Reply:
oh interesting, I was meant to link to https://www.uvicorn.org/server-behavior/#timeouts but somehow got the wrong one. Thanks for catching it!